### PR TITLE
Added support for Card Metadata API (CS2)

### PR DESCRIPTION
**Pull request by Armando-Ibarra**

### CODE REVIEW

**Review by armandojaleo — APPROVED** [review]
💯

File: lib/Checkout/Metadata/MetadataClient.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+use Checkout\CheckoutApiException;
+use Checkout\CheckoutConfiguration;
+use Checkout\Client;
+use Checkout\Metadata\Card\CardMetadataRequest;
+
+class MetadataClient extends Client
+{
+    public function __construct(ApiClient $apiClient, CheckoutConfiguration $configuration)
+    {
+        parent::__construct($apiClient, $configuration, AuthorizationType::$oAuth);
```
Comment (martinseco) [inline lib/Checkout/Metadata/MetadataClient.php:16]: Shouldn't `AuthorizationType` be `secretKeyOrOAuth`?

**Review by martinseco — CHANGES_REQUESTED** [review]
Looks good! Just one comment about AuthorizationType. If it's correct this way, tell me and I'll approve it

File: lib/Checkout/Metadata/MetadataClient.php (hunk 1 above)
Comment (Armando-Ibarra) [reply lib/Checkout/Metadata/MetadataClient.php:16]: Yes you're right good catch

**Pull request merged**